package.json

```diff
@@ -26,10 +25,7 @@
     "test:cov": "jest --silent --coverage",
     "lint": "prettier . --check ; eslint '*.ts' --quiet --fix",
     "lint:fix": "prettier . --write ; eslint . --fix",
-    "acceptance": "npm run acceptance-tests -w ui-automation-tests --",
-    "accessibility": "npm run accessibility-tests -w ui-automation-tests --",
```

Comment:
I see these have moved out to ui-automation, but aren't run in the script. When were these run before?

Reply:
Sorry, I meant the a11y tests specifically, not the acceptance ones